File: Exercises/C++/Dietel & Dietel C++ How to Program/Chapter_5/Examples/SumEven/SumEven/sum.cpp
//Fig. 5.5: fig05-05.cpp
//Summing integers with the for statememt

#include <iostream>
using namespace std;

int main()
{
	int total = 0; // initialize total

	// total even integers from 2 through 20
	for (int number = 2; number <= 20; number += 2)
		total += number;

	cout << "Sum is " << total << endl; // display results
} // end main

File: Exercises/Java/Java_How to Program_8th Edition/Visual Studio/Chapter 5_Control Statements Part2/Examples/Sum/Sum/src/main/java/pkg/Sum.java
package pkg;

// Fig. 5.5: Sum
// Summing integers with the for statement.

public class Sum 
{
	public static void main(String[] args) 
	{
		int total = 0; // initialize total

		// total even integers from 2 through 20
		for (int number = 2; number <= 20; number += 2)
			total += number;

		System.out.printf("Sum is %d\n", total); // display results
	} // end main
} // end class Sum
